cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala

```diff
@@ -97,6 +97,9 @@ object Wadl2Dot {
   val help = parser.flag[Boolean] (List("h", "help"),
                                    "Display usage.")
 
+  val xpathVersion = parser.option[Int](List("t", "xpath-version"), "n",
```

Comment:
Should I make the help comment state that it isn't used, but is only available for consistency across all the tools?